ui/src/components/btn/QasBtn.vue

```diff
@@ -76,12 +86,17 @@ export default {
         ...(this.showLabel && { label: this.label }),
 
         ...attributes,
+        icon: this.icon,
+        iconRight: this.iconRight,
         class: [this.classes, externalClass]
       }
     },
 
     hasIconOnly () {
-      return (!this.label || !this.showLabel) && (this.$attrs.icon || this.$attrs.iconRight)
+      return (
+        (!this.label || !this.showLabel) &&
+        ((this.icon && !this.iconRight) || (this.iconRight && !this.icon))
```

Comment:
Deu uma bugada na minha mente aqui, mas essa condição não seria a mesma que: `this.icon || this.iconRight`? 🤔